spec/current_ability_spec.rb

```diff
@@ -136,15 +138,22 @@
   end
 
   describe "caching" do
-    before(:each) do
+    before do
```

Comment:
Spec to ensure nested blocks work in abilities. We are not caching this way anymore but the test seemed like a good one to leave.